src/python/perfflowaspect/advice_chrome.py

```diff
@@ -150,6 +154,14 @@ class ChromeTracingAdvice:
         os.makedirs(log_dir, exist_ok=True)
         fn = os.path.join(log_dir, fn)
 
+    log_enable = perfflow_options["log-enable"]
+    if log_enable in ["True", "true", "TRUE"]:
```

Comment:
Maybe also add "T" and "1" here.

Reply:
I think adding `T` and 1 as options in python is straight-forward, but on the C side, it's looking for strings. Does it make sense to use `1` as a string here?